core/src/main/java/it/pagopa/selfcare/party/registry_proxy/core/StationServiceImpl.java

```diff
@@ -39,7 +39,8 @@ public QueryResult<Station> search(Optional<String> searchText, int page, int li
     public Station findByTaxId(String taxId) {
         log.trace("find SA By taxId start");
         log.debug("find SA By taxId = {}", taxId.toUpperCase());
-        final List<Station> pdndList = indexSearchService.findById(Station.Field.TAX_CODE, taxId.toUpperCase());
+        taxId = "ANAC_" + taxId;
```

Comment:
you could introduce a constant class for string "ANAC_", for ex. ANAC_PREFIX = "ANAC_"....it's better avoiding to use hardcoded string